src/storage/src/read.rs

```diff
@@ -38,7 +38,7 @@ pub struct Batch {
     ///
     /// Columns follow the same order convention of region schema:
     /// key, value, internal columns.
-    columns: Vec<VectorRef>,
+    pub columns: Vec<VectorRef>,
```

Comment:
Do we need to pub this? We have a `columns()` method

Reply:
Currently I need to take the columns from Batch to construct a Chunk without cloning VectorRef